cmd/skaffold/app/cmd/init.go

```diff
@@ -133,14 +136,41 @@ func doInit(out io.Writer) error {
 	if err != nil {
 		return err
 	}
-	if outfile != "" {
-		if err := ioutil.WriteFile(outfile, cfg, 0644); err != nil {
-			return errors.Wrap(err, "writing config to file")
-		}
-	} else {
+
+	if opts.ConfigurationFile == "-" {
 		out.Write(cfg)
+		return nil
 	}
 
+	if !force {
+		fmt.Fprintln(out, string(cfg))
+
+		reader := bufio.NewReader(os.Stdin)
+	confirmLoop:
+		for {
+			fmt.Fprintf(out, "Do you want to write this configuration to %s? [y/n]: ", opts.ConfigurationFile)
```

Comment:
this is breaking integration tests - we need a non-interactive way (-y):
````
--- FAIL: TestInit/microservices (0.02s)
    	run_test.go:352: running init: Running [skaffold init -f skaffold.yaml.out -a leeroy-app/Dockerfile=gcr.io/k8s-skaffold/leeroy-app -a leeroy-web/Dockerfile=gcr.io/k8s-skaffold/leeroy-web]: stdout apiVersion: skaffold/v1alpha4
    		kind: Config
    		build:
    		  artifacts:
    		  - image: gcr.io/k8s-skaffold/leeroy-app
    		    context: leeroy-app
    		  - image: gcr.io/k8s-skaffold/leeroy-web
    		    context: leeroy-web
    		deploy:
    		  kubectl:
    		    manifests:
    		    - leeroy-app/kubernetes/deployment.yaml
    		    - leeroy-web/kubernetes/deployment.yaml
    		Do you want to write this configuration to skaffold.yaml.out? [y/n]: , stderr: time="2018-10-04T07:51:21Z" level=fatal msg="reading user confirmation: EOF"
    		, err: exit status 1: exit status 1, output: apiVersion: skaffold/v1alpha4
    		kind: Config
    		build:
    		  artifacts:
    		  - image: gcr.io/k8s-skaffold/leeroy-app
    		    context: leeroy-app
    		  - image: gcr.io/k8s-skaffold/leeroy-web
    		    context: leeroy-web
    		deploy:
    		  kubectl:
    		    manifests:
    		    - leeroy-app/kubernetes/deployment.yaml
    		    - leeroy-web/kubernetes/deployment.yaml
    		Do you want to write this configuration to skaffold.yaml.out? [y/n]:
    	run_test.go:342: error removing generated skaffold yaml: remove ../examples/microservices/skaffold.yaml.out: no such file or directory
````

Reply:
There's a `--force` that I forgot to use here.